### DISCUSSION TITLE
Mobile Development OR Web Development?

### DISCUSSION THREAD

**Discussion by fcfcnb** [drknzz/GitHub-Achievements#277] [General]
#res

**Comment by mesheryar** [comment]
Web Development

**Comment by AamirMir1** [comment]
mobile development

**Comment by the1Riddle** [comment]
depends

**Comment by MorrisrCodes** [comment]
web development is pretty saturated.

**Comment by viniarski** [comment]
Depends of the region:
https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide

**Comment by coreybailey07** [comment]
I'd say web development. You will get your feet underneath you with HTML, CSS and JavaScript. As you become familar with frontend frameworks, that knowledge will transfer into mobile app frameworks such as React Native, Flutter or Xamarin. This way you'll will be contributing mobile developer that can later focus on a specific native language (Swift, Java, C#) to take your career to the next level.

**Comment by npmRunTheWorld** [comment]
Here's the trick... why not both? It depends on what you really want to specialize in. But if you want cross platform capabilities look into react native build something for web, mobile, etc. If you want to specialize in web alone which can technically translate to most platforms because all devices have a browser these days. Id suggest learning Next.JS (react), or Nuxt 3 (vue 3), and get more into JavaScript. If you like this answer make sure to accept the answer for the discussion!

**Comment by Saurabh-S-Patil** [comment]
### **Mobile Development:**

- Focuses on creating applications for mobile devices such as smartphones and tablets.
- Requires knowledge of specific mobile platforms like iOS (Swift) or Android (Java/Kotlin).
- Typically involves designing user interfaces optimized for smaller screens and touch interactions.
- Often involves utilizing platform-specific features like push notifications and geolocation.

### **Web Development:**

- Involves creating websites or web applications accessed via web browsers.
- Requires proficiency in web technologies such as HTML, CSS, and JavaScript.
- Can include front-end development for user interface design and back-end development for server-side logic.
- Requires considerations for cross-browser compatibility and responsive design for various screen sizes.
